src/test/ui/issues/issue-22560.stderr

```diff
@@ -1,34 +1,42 @@
 error[E0393]: the type parameter `Rhs` must be explicitly specified
-  --> $DIR/issue-22560.rs:3:13
+  --> $DIR/issue-22560.rs:6:13
    |
-LL | type Test = Add +
+LL |             Sub;
    |             ^^^ missing reference to `Rhs`
    |
    = note: because of the default `Self` reference, type parameters must be specified on object types
 
 error[E0393]: the type parameter `Rhs` must be explicitly specified
-  --> $DIR/issue-22560.rs:6:13
+  --> $DIR/issue-22560.rs:3:13
    |
-LL |             Sub;
+LL | type Test = Add +
    |             ^^^ missing reference to `Rhs`
    |
    = note: because of the default `Self` reference, type parameters must be specified on object types
 
 error[E0225]: only auto traits can be used as additional traits in a trait object
   --> $DIR/issue-22560.rs:6:13
    |
+LL | type Test = Add +
+   |             --- first non-auto trait
+...
 LL |             Sub;
-   |             ^^^ non-auto additional trait
+   |             ^^^ additional non-auto trait
 
-error[E0191]: the value of the associated type `Output` (from the trait `std::ops::Add`) must be specified
+error[E0191]: the value of the associated types `Output` (from the trait `std::ops::Add`), `Output` (from the trait `std::ops::Sub`) must be specified
   --> $DIR/issue-22560.rs:3:13
    |
 LL |   type Test = Add +
    |  _____________^
+   | |_____________|
+   | |
 LL | |
 LL | |
 LL | |             Sub;
-   | |_______________^ associated type `Output` must be specified
+   | |               ^
+   | |_______________|
+   | |_______________associated type `Output` must be specified
+   |                 associated type `Output` must be specified
```

Comment:
Can you make these spans point just at `Add` and `Sub`? (preexisting)

Reply:
This was actually a pre-existing issue, but yes, definitely needs to be fixed.

Reply:
Okay, it turns out this isn't so easy... there is no straightforward way to find the trait usage/span a missing associated type belongs to, so that's why right now the span of the entire trait object is used. @Centril and I discussed this, and we think the current diagnostic is actually more correct, though the multiline rendering of it makes it look a bit awkward.

CC @estebank for future improvement.

Reply:
can you open an issue so it doesn't get lost?

Reply:
Sure, will do. Are you satisfied with everything else now?

Reply:
https://github.com/rust-lang/rust/issues/60595